README.md

```diff
@@ -1 +1,137 @@
-# Microverse-mini-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>

```

Comment:
- [ ] Kindly remove this logo link since the logo file doesn't exist it's showing a "404 page not found" error and this makes your project look unprofessional nad a bit reckless so If you don't intend on putting your own logo you can remove it.